internal/services/dispatch/dispatchService.go

```diff
@@ -101,19 +102,21 @@ func (s *dispatchService) SubscribeForDispatch(ctx context.Context, email, dispa
 	return nil
 }
 
+var TemplateParseErr = errors.New("template error")
+
 func (s *dispatchService) parseHTMLTemplate(templateName string, data any) ([]byte, error) {
 	templateFile := mailing.PathToTemplate(templateName + ".html")
 	tmpl, err := template.ParseFiles(templateFile)
 	if err != nil {
 		s.log.Errorf("failed to parse html template %s: %v", templateFile, err)
 
-		return nil, err
+		return nil, fmt.Errorf("%w: %w", TemplateParseErr, err)
```

Comment:
Just use errors.Wrap

Reply:
What is the benefit of errors.Wrap?

Reply:
I guess yesterday i've answered on your question?

Reply:
Holy crap, sry.
The std package naming is errros.Join and errors.Unwrap)
Just use join

Reply:
Yep, I knew about it :)
Replaced errors.Wrap with errros.Join